metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -398,28 +397,42 @@ def _get_bigquery_log_entries(
         template: str,
     ) -> Union[Iterable[AuditLogEntry], Iterable[BigQueryAuditMetadata]]:
         # Add a buffer to start and end time to account for delays in logging events.
+        start_time = (self.config.start_time - self.config.max_query_duration).strftime(
+            BQ_DATETIME_FORMAT
+        )
+        self.report.start_time = start_time
```

Comment:
The config already has a start_time. This is actually the query_start_time. The report could confuse users, if max_query_duration is set. Please rename it to `log_entry_query_start_time`.

Reply:
will fix to rename these properly to avoid confusion